kolibri/plugins/management/assets/src/device_management/views/manage-content-page/index.vue

```diff
@@ -1,9 +1,11 @@
 <template>
 
   <div>
-
     <template v-if="canManageContent">
-      <component v-if="pageState.wizardState.shown" :is="wizardComponent" />
+      <component
+        v-if="wizardPageName!==''"
```

Comment:
we conventionally add whitespace around the operator - this comes up in a few places in templates:

`v-if="wizardPageName !== ''"`